src/khoj/database/models/__init__.py

```diff
@@ -234,9 +234,11 @@ class ModelType(models.TextChoices):
 class TextToImageModelConfig(BaseModel):
     class ModelType(models.TextChoices):
         OPENAI = "openai"
+        STABILITYAI = "stability-ai"
 
     model_name = models.CharField(max_length=200, default="dall-e-3")
     model_type = models.CharField(max_length=200, choices=ModelType.choices, default=ModelType.OPENAI)
+    api_key = models.CharField(max_length=200, default=None, null=True, blank=True)
```

Comment:
Field is not being used for OpenAI models.

Reply:
Fixed that. Field is now used when set